docs/index.md

```diff
@@ -45,72 +43,122 @@ Download the latest version of the CLI for your operating system and make sure t
+To measure the performance of your team, you must send information to Pulse about the following key events whenever they happen in the software delivery workflow of your primary application or service:
 
--  [Deployments](#deployments)
 -  [Changes](#changes)
+-  [Deployments](#deployments)
 -  [Incidents](#incidents)
 
 !!! important
-    Before setting up the integration with Pulse, make sure that you have an API key that identifies your organization and authorizes you to send data to Pulse.
+    Before setting up the integration with Pulse, make sure that you have an API key provided by Pulse to identify your organization and authorize you to send data to Pulse.
 
-#### Deployments
+### Changes
 
-**For SaaS applications,** send information to Pulse whenever you deploy to production.
+Report an event to Pulse whenever your team commits a change to a repository.
 
-**For self-hosted applications,** a better option might be to send information to Pulse whenever you have a valid artifact ready to be delivered to any user.
+Pulse uses these reports to calculate the metric [Lead time for changes](metrics.md#lead-time-for-changes).
 
-| Field      | Description                                            | Format                                       |
-| ---------- | ------------------------------------------------------ | -------------------------------------------- |
-| identifier | Version or another unique identifier of the deployment | String                                       |
-| timestamp  | Time when the deployment finished                      | Number<br/>(Unix epoch timestamp in seconds) |
-|            | Commit identifiers included in the deployment          | String<br/>(space-separated list)            |
+You must send the following information when reporting changes to Pulse:
+
+| Field      | Description                                             | Format                                       |
+| ---------- | ------------------------------------------------------- | -------------------------------------------- |
+| identifier | The commit identifier                                   | String                                       |
+| timestamp  | Time when the commit was first pushed to the repository | Number<br/>(Unix epoch timestamp in seconds) |
+
+Using the Pulse CLI:
 
 ```sh
-./event-cli push deployment \
+./event-cli push change \
     --api-key "<API KEY>" \
-    --identifier "<deployment identifier>" \
-    --timestamp "$(date +%s)" \
-    <space-separated list of commit identifiers>
+    --identifier "<change identifier>" \
+    --timestamp "<timestamp>"
 ```
 
-<!-- IDEA
-     Consider including example snippets for the webhook -->
+Using the webhook:
 
-#### Changes
+```sh
+curl -X POST https://ingestion.acceleratedevops.net/v1/ingestion/<PROVIDER>
+    -d "api_key=<API KEY>"
+    -d "identifier=<change identifier>"
+    -d "timestamp=<timestamp>"
+```
```

Comment:
I know that right now the exact syntax to call the webhook is not well-defined. But please check if this makes sense.

For now, this is meant more as guidance to give confidence to users that using the webhook is relatively straightforward.

Reply:
IMHO this makes no sense. For webhooks we expect to receive the payload from services like GitHub not a manual format. As mentioned earlier we were only expecting to document this as we implement providers which rn we do not have any.